resources/sdk/webmessagingjava/extensions/ConsoleColors.java

```diff
@@ -0,0 +1,81 @@
+package cloud.genesys.webmessaging.sdk;
+
+// Inspired by https://stackoverflow.com/a/45444716/1124338
+public class ConsoleColors {
```

Comment:
This class isn't/shouldn't be used anywhere and should be removed. I only had it in there originally because I was using it during development to aid local logging and debugging. The SDK should use the slf4j logger to emit trace messages.